conf/application.conf

```diff
@@ -115,6 +115,11 @@ app {
     ttl = ${IZANAMI_PASSWORD_RESET_REQUEST_TTL}
     ttl = 900
   }
+  # Modify the score related to the similarity of the search query
+  search {
+    parameter = ${?IZANAMI_SEARCH_PARAMETER}
```

Comment:
"parameter" is not a very clear name, something like "similarity-threshold" would be more explicit.

Reply:
Ok,  I will change it to "similarity-threshold".